pkg/cmd/run/view/view.go

```diff
@@ -93,14 +94,17 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 	cmd := &cobra.Command{
 		Use:   "view [<run-id>]",
 		Short: "View a summary of a workflow run",
-		Args:  cobra.MaximumNArgs(1),
+		Args:  cobra.MaximumNArgs(2),
```

Comment:
this doesn't need to change. `run view` is still accepting only 1 argument (a run identifier). Adding a new flag doesn't count toward arg count.